.vscode/settings.json

```diff
@@ -2,13 +2,16 @@
   "editor.tabSize": 2,
   "files.eol": "\n",
   "typescript.tsdk": ".yarn/sdks/typescript/lib",
+  "typescript.tsserver.watchOptions": {
+    "watchFile": "useFsEventsOnParentDirectory",
+    "watchDirectory": "useFsEvents"
+  },
   "npm.packageManager": "yarn",
   "eslint.validate": [
     "javascript",
     "typescript",
     "typescriptreact"
   ],
-  "eslint.enable": true,
```

Comment:
Accidental I assume

Reply:
No, when I opened the settings file this one was flagged as deprecated 🤔

Reply:
![image](https://user-images.githubusercontent.com/1037931/96871028-fe957600-1471-11eb-9391-ccdd3aec7cc9.png)

Reply:
Ah, good to know